v3/newrelic/config.go

```diff
@@ -401,21 +407,34 @@ type Config struct {
 		// names look like they are internal to the agent itself.
 		IgnoredPrefixes []string
 	}
+
+	// ModuleDependencyMetrics controls reporting of the packages used to build the instrumented
+	// application, to help manage project dependencies.
+	ModuleDependencyMetrics struct {
+		// Enabled controls whether the module dependencies are collected and reported.
+		Enabled bool
+		// RedactIgnoredPrefixes, if true, redacts a non-nil list of IgnoredPrefixes from
+		// the configuration data transmitted by the agent.
+		RedactIgnoredPrefixes bool
```

Comment:
IMO this should just be the default behavior, or default to true, especially since there isn't an easy way to set this in the config_options. Alternatively, if the customer is running in high security mode, we could always set this to true.

Reply:
Oops. I meant to provide options to set those. I also agree that making them on by default makes sense.